check_diff/tests/git.rs

```diff
@@ -0,0 +1,34 @@
+use check_diff::clone_git_repo;
+
+use std::fs;
+use tempfile::Builder;
+use tracing::error;
+
+#[test]
+fn clone_repo_test() {
+    let dir = Builder::new().tempdir_in("");
```

Comment:
Similar comment as before. It's probably simpler to use the [tempdir](https://docs.rs/tempfile/latest/tempfile/fn.tempdir.html) function and just `unwrap` if we fail to create a tempdir.